### ISSUE TITLE
ChallengeEditor container - Remove potentially unused code

### ISSUE THREAD

**Issue by vikasrohit** [topcoder-platform/challenge-engine-ui#708]
<img width="1346" alt="Screen Shot 2020-06-30 at 10 33 16 AM" src="https://user-images.githubusercontent.com/2417632/86085589-232da280-babd-11ea-86a7-2e6f8316999b.png">
Seems like we have this piece of code in last major refactoring https://github.com/topcoder-platform/challenge-engine-ui/commit/4b9d810816767489a262eb9d819aaf79453e50a2#diff-41ccc585907d351fe4b6fb36fccc24f1R48-R56. It does not seem to be a good way (it is loading challenge details again on page unload) to implement any feature anyway, so we have to identify the feature or bug for which this piece of code is used and then try to fix it with more robust solution.

**Comment by vikasrohit** [comment]
I have marked it as P1 because it this challenge loading is making debugging more difficult as state of applications is not predictable.

**Comment by vikasrohit** [comment]
@drasticdpk I am not sure what exact regression, removing this code, would bring but I am suspecting some kind of non blocking issue to appear with this change. I tried to find that issue since last two weeks on my local (with this code being removed) but I was not able to locate a specific issue which can appear because of this change. All of the issues that I encountered in this duration are also happening with this piece of code as well. Hence, I have decided to remove this code for now and see if it causes any regression and fix that regression in more appropriate way.
For reference, originally this code was introduced for https://github.com/topcoder-platform/challenge-engine-ui/issues/659
